src/main/flight/setpoint.c

```diff
@@ -200,6 +208,10 @@ void setpointUpdate(void)
 
         SP = sp.setpoint[axis] = applyRatesCurve(axis, SP);
         DEBUG_AXIS(SETPOINT, axis, 4, SP);
+
+        // Apply boost
+        SP = sp.setpoint[axis] +=
+            difFilterApply(&sp.boostFilter[axis], SP) * sp.boostGain[axis];
```

Comment:
After this line, we should really have `DEBUG_AXIS(SETPOINT, axis, 5, SP);`
Need to change current use of 5 and 6 to 6 and 7.

Reply:
Why not just 7? Doesn't seem very manageable to change the existing ones. Also, we will soon add another for the dynamic deadzone.